lgtm.yaml

```diff
@@ -0,0 +1,36 @@
+path_classifiers:
+  docs:
+    - doc
+    - exclude: doc/cli
+  test:
+    - libvast_test
+    - integration
+    - libvast/test
+  template:
+    - scripts
+extraction:
+  cpp:
+    prepare:
+      packages:
+        - libssl-dev
+        - python-pip
+    after_prepare:
+      - mkdir -p $LGTM_WORKSPACE/latest-gcc-symlinks
+      - ln -s /usr/bin/g++-8 $LGTM_WORKSPACE/latest-gcc-symlinks/g++
+      - ln -s /usr/bin/gcc-8 $LGTM_WORKSPACE/latest-gcc-symlinks/gcc
+      - export BUILD_TYPE=Release
```

Comment:
Should this be Debug so it looks at all the codepaths that are hidden behind `#if VAST_LOG_LEVEL >= VAST_LOG_LEVEL_DEBUG` as well?

Reply:
As discussed with the team, we'll go with `Release` for now, as there is no important logic to be conditionally compiled in `Debug` builds. `Debug` affects mostly logging behavior.